client/client.go

```diff
@@ -64,7 +64,7 @@ func (c *Client) CompletedObjectives() <-chan protocols.ObjectiveId {
 // CreateDirectChannel creates a directly funded channel with the given counterparty
 func (c *Client) CreateDirectChannel(counterparty types.Address, appDefinition types.Address, appData types.Bytes, outcome outcome.Exit, challengeDuration *types.Uint256) protocols.ObjectiveId {
 	// Convert the API call into an internal event.
-	objective, _ := directfund.New(true,
+	objective, _ := directfund.NewObjective(true,
```

Comment:
Here we consume the constructor from a different package. This is part of the motivation for the renaming done in thiss PR.